### DISCUSSION TITLE
更新后输出失败问题

### DISCUSSION THREAD

**Discussion by Frank2jiang** [Tencent/libpag#1856] [Q&A]
输出没有错误显示，但是输出到100%就是会一个叉号，尝试了旧版本输出，是可以输出的
![4ddfff6cf62797c7dc3da6e3499affe8-screenshots](https://github.com/Tencent/libpag/assets/150412079/f41aac0d-54f3-423e-9b8d-14e97b89e2be)

版本4.0.5（stable）

**Comment by kevingpqi123** [comment]
将AE 工程附上，然后附上平台、系统版本等，仅仅上面的信息无法帮助到你

**Comment by kevingpqi123** [comment]
长时间无回复，暂时关闭
